package.json

```diff
@@ -43,9 +43,9 @@
     "webpack-merge": "^4.1.4"
   },
   "dependencies": {
-    "@material-ui/core": "^3.9.0",
+    "@material-ui/core": "^4.0.0-rc.0",
```

Comment:
should be `4.1.3` (latest at the time of writing), not rc

Reply:
Done